bindings/c/test/apitester/TesterApiWorkload.cpp

```diff
@@ -380,56 +380,24 @@ void ApiWorkload::setupBlobGranules(TTaskFct cont) {
 void ApiWorkload::blobbifyTenant(std::optional<int> tenantId,
                                  std::shared_ptr<std::atomic<int>> blobbifiedCount,
                                  TTaskFct cont) {
-	auto retBlobbifyRange = std::make_shared<bool>(false);
 	execOperation(
 	    [=](auto ctx) {
 		    fdb::Key begin(1, '\x00');
 		    fdb::Key end(1, '\xff');
 
 		    info(fmt::format("setup: blobbifying {}: [\\x00 - \\xff)\n", debugTenantStr(tenantId)));
 
-		    fdb::Future f = ctx->dbOps()->blobbifyRange(begin, end).eraseType();
-		    ctx->continueAfter(f, [ctx, retBlobbifyRange, f]() {
-			    *retBlobbifyRange = f.get<fdb::future_var::Bool>();
+		    // wait for blobbification before returning
+		    fdb::Future f = ctx->dbOps()->blobbifyRange(begin, end, true).eraseType();
```

Comment:
We should probably test both true and false case

Reply:
Yeah we should. This is less of a test than just for setting up the blob data though, where this parameter of true replaces the blobbify+verify behavior.

Reply:
Yes true. Maybe we need to add blobbifyRange into the operation mix of an existing workload. Just need to check if it does not break the assumptions of other operations. Otherwise maybe it would be worth introducing a separate test. The choice between wait and no wait could be picked randomly

Reply:
Yeah I had thought about introducing a test somewhat like the simulation BlobGranuleRangesWorkload to the api tester, that would cover that case and others that we're lacking in the API tester for the blob granule management functions

Reply:
Then let's do it in a separate PR